source/octf/proto/parsedTrace.proto

```diff
@@ -57,7 +58,7 @@ message ParsedEvent {
 
     message FileInfo {
         /** File ID */
-        uint64 id = 1;
+        EventNodeId id = 1;
```

Comment:
I'd expect in output uint64 file.id and string or uint64 file.createTime. So let's have here two fields

Reply:
With google's timestamp the creation time has a more human readable format:
{"fileId":"103662261","creationDate":"2020-03-10T09:34:23.087201157Z"}